--- kernel/nvidia/nv-caps.c	2024-07-16 14:46:45.508301417 -0700
+++ kernel-new/nvidia/nv-caps.c	2024-07-16 14:54:00.612064766 -0700
@@ -600,7 +600,7 @@ void NV_API_CALL nv_cap_close_fd(int fd)
     close_fd(fd);
 #elif NV_IS_EXPORT_SYMBOL_PRESENT___close_fd
     __close_fd(current->files, fd);
-#if (LINUX_VERSION_CODE < KERNEL_VERSION(4, 17, 0))
+#elif (LINUX_VERSION_CODE < KERNEL_VERSION(4, 17, 0))
     sys_close(fd);
 #else
     ksys_close(fd);
